src/clib-install.c

```diff
@@ -399,7 +399,7 @@ int main(int argc, char *argv[]) {
   }
 
   if (!root_package) {
-    char *name = NULL;
+    const char *name = NULL;
     char *json = NULL;
```

Comment:
Why `name` is const and `json` isn't?

Reply:
`json` is assigned the result of `fs_read`. The return type of `fs_read` is `char *`, which would be inconsistent.

Reply:
🤦 I just saw the function signature at the top.